meta-networking/recipes-support/libconfuse/libconfuse_3.3.bb

```diff
@@ -0,0 +1,27 @@
+DESCRIPTION = "libConfuse is a configuration file parser library"
+LICENSE = "CUSTOM"
+LIC_FILES_CHKSUM = "file://LICENCE;md5=42fa47330d4051cd219f7d99d023de3a"
+
+DEPENDS = "gettext-native"
+
+#SRCREV = "1b3f11bde315e221474f7d066ce4efb4ff4d39e3"
+SRC_URI = "git://github.com/libconfuse/libconfuse.git;branch=master;tag=v3.3"
+
+S = "${WORKDIR}/git"
+
+inherit autotools pkgconfig
+
+do_configure_prepend(){
+    cd ${S}
+    ${S}/autogen.sh
+}
+
+do_compile(){
+    cd ${S}
+    oe_runmake
+}
```

Comment:
you can inherit `autotooks-brokensep` instead of `autotools` if the package is not buildable out of tree. and then you wont need to override `do_compile` and `do_intall`

Reply:
Ok, I only kept do_configure, the other worked. Because I use an older version of oe, I did not use autotooks-brokensep but only B = "${S}".